templates/default/flyingduck.service.erb

```diff
@@ -21,7 +21,7 @@ ExecStart = /usr/bin/docker run --rm --name %n \
             --memory_limit=${FLYINGDUCK_MEMORY} \
             --request_queue_timeout=<%= node['flyingduck']['request_timeout_sec'] %> \
             --query_result_batch_size=<%= node['flyingduck']['batch_size'] %> \
-            --fqdn=<%= @flyingduck_fqdn %> \
+            --fqdn=0.0.0.0 \
```

Comment:
My only concern with this change is that this property is used both for the bind address of Arrow server but also to create the endpoint clients connect to https://github.com/logicalclocks/flyingduck/blob/main/src/server.py#L208

Is this still how you would use it https://github.com/logicalclocks/flyingduck/blob/main/example/client.ipynb?short_path=1b815a6#L95 ? If yes, then it might work from localhost, it's OS dependent to translate connecting to 0.0.0.0 to connecting to 127.0.0.1, but it won't work from anywhere else.

Reply:
I didn't write this code, cant claim to understand it.
But, my observed behaviour is that if you have `--fqdn` bind to the consul name, outside access doesn't work. This is nothing to do with localhost.
That this code is used to create the endpoint clients connect to is why the consul name is broken. Clients outside hopsworks don't know about consul domain names.

Reply:
Well if it works, then I don't have any comment :)